operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -131,8 +131,8 @@ groups:
         Number of members in Mimir compactor hash ring does not match the expected number in {{ $labels.cluster }}/{{ $labels.namespace }}.
     expr: |
       (
-        avg by(cluster, namespace) (sum by(cluster, namespace, pod) (cortex_ring_members{name=~"compactor.*|cortex|mimir|mimir-backend",job=~".*/compactor"}))
-        != sum by(cluster, namespace) (up{job=~".*/compactor"})
+        avg by(cluster, namespace) (sum by(cluster, namespace, pod) (cortex_ring_members{name=~"compactor.*|cortex|mimir|mimir-backend",job=~"(.*/)?compactor.*|cortex|mimir|mimir-backend"}))
+        != sum by(cluster, namespace) (up{job=~"(.*/)?compactor.*|cortex|mimir|mimir-backend"})
```

Comment:
Ops, regex `|` precedence looks wrong here. Can you open a dedicated PR to fix all `_config.job_names` and make sure they're all wrapped by `()`? Few of them aren't and looks a bug.

Reply:
Ah, good catch. I did however notice `_config.job_names` is a bit odd in that one of the entries is a list (not a regex). I could look at fixing that, or I could change it here to something like `(.*/)?(%(job)s)`. Thoughts?

Reply:
Ignore the list. Its regex should be built correctly. My comment is just about the fields where we write the regex directly.